translations/fr/try_ruby_160.md

```diff
@@ -0,0 +1,27 @@
+---
+lang:   EN
+title:  Sauvegardé, rangé proprement
+answer: \[12, 35, 47\]
+load:   ticket = [12, 47, 35]
+ok:     Tu as organisé la liste en ordre
+error:
+---
+
+Tu as gardé tes numéro de lotto en le rangeant dans une __variable__
+nommée ticket. Mais n'importe quel autre nom, comme _lalistedejimmy_,
+aurait aussi fait l'affaire.
+
+Comme tu découvrira, la programmation c'est surtout jongler des
```

Comment:
What do you think about using `jongler avec` instead of `jongler`?

Reply:
👍